boa/src/vm/instructions.rs

```diff
@@ -57,6 +57,10 @@ pub enum Instruction {
     Neg,
     BitNot,
     Not,
+
+    DefVar(usize, usize),
+    DefLet(usize, usize),
+    DefConst(usize, usize),
```

Comment:
Could we add some documentation/comments to these? So others know what the 2 usize arguments are for
As a bonus putting these in some alphabetical order would be nice (but not essential)

Reply:
We could even use attributes in a struct-like variant, with names. Would make things much more clear.